app_dart/lib/src/request_handlers/github_webhook.dart

```diff
@@ -370,7 +339,7 @@ class GithubWebhook extends RequestHandler<Body> {
     await buildBucketClient.batch(BatchRequest(requests: requests));
   }
 
-  Future<bool> _isIgnoredForGold(PullRequestEvent event) async {
+  Future<bool> _isIgnoredForGold(String eventAction, PullRequest event) async {
     bool ignored = false;
     String rawResponse;
     try {
```

Comment:
Is this something we can get directly from event?

Reply:
It was eventAction at the call to _isIgnoredForGold

Reply:
No, eventAction is not a field.